_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -15,6 +15,12 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  using-aria:fourth_rule_of_aria_use:
+    title: Fourth rule of ARIA use
+    forConformance: true
```

Comment:
This is not a conformance document. This should be `false`

Reply:
OK.

I'm not sure I fully understand what "conformance document" is. Notably because currently the rules that map to ARIA 1.1 are "required for conformance" and I am not sure this is a conformance document either.
See https://act-rules.github.io/rules/6a7281 for example.

Reply:
Actually, I think that "Fourth rule of ARIA use" qualifies as a "accessibility requirements" because it "_has to be satisfied in order to conform to a standard_" where the standard is "Using ARIA", and it is "_aimed at improving access for people with disabilities to an ICT product._"

So it does fit in the definition provided by the ACT rules format.

Reply:
That's not correct. This document has no normative requirements in it, just like the techniques:

> This document is informative only. Resources are for information purposes only, no endorsement implied.